src/components/ListingDetails/index.tsx

```diff
@@ -0,0 +1,320 @@
+  },
+  // Client Location
+  {
+    label: 'Client Location',
+    getValue: (data: Listing) => (data as CaseListing).client_location || 'N/A',
+  },
+  // Custody Location
+  {
+    // Temporary
+    label: 'Custody Location',
+    getValue: () => 'San Diego, CA',
+  },
+];
+
+const caseInterpretationFields = [
+  // Time Commitment
+  {
+    label: 'Time Commitment',
+    getValue: (data: Listing) =>
+      parseTimeCommitment(
+        (data as CaseListing).hours_per_week,
+        (data as CaseListing).num_weeks,
+      ),
+  },
+  // Languages
+  {
+    label: 'Language(s)',
+    getValue: (data: Listing) => (data as CaseListing).languages.join(', '),
+  },
+  // Remote/In Person
+  {
+    label: 'Remote/In Person',
+    getValue: (data: Listing) =>
+      (data as CaseListing).is_remote ? 'Remote' : 'In Person',
+  },
+];
+
+const intepretationFields = [
+  // Languages
+  {
+    label: 'Language(s)',
+    getValue: (data: Listing) => (data as Interpretation).languages.join(', '),
+  },
+  // Interpretation Type
+  {
+    label: 'Interpretation Type', // 'Language Support Type' for consistency w/ Doc?
+    getValue: (data: Listing) => 'One-Time Intepretation',
+  },
+  // Remote/In Person
+  {
+    label: 'Remote/In Person',
+    getValue: (data: Listing) =>
+      (data as Interpretation).is_remote ? 'Remote' : 'In Person',
+  },
+];
+
+const docFields = [
+  // Languages
+  {
+    label: 'Language(s)',
+    getValue: (data: Listing) =>
+      (data as DocumentTranslation).languages.join(', '),
+  },
+  // Language Support Type
+  {
+    label: 'Language Support Type',
+    getValue: (data: Listing) => 'Written',
+  },
+  // Number of Pages
+  {
+    label: 'Number of Pages',
+    getValue: (data: Listing) => (data as DocumentTranslation).num_pages,
+  },
+];
+
+const LCAFields = [
+  // Country Field
+  {
+    label: 'Country Field',
+    getValue: (data: Listing) => (data as LimitedCaseAssignment).country,
+  },
+  // Language(s)
+  {
+    label: 'Language(s)',
+    getValue: (data: Listing) =>
+      (data as LimitedCaseAssignment).languages.join(', '),
+  },
+  // Expected Deliverable
+  {
+    label: 'Expected Deliverable',
+    getValue: (data: Listing) => (data as LimitedCaseAssignment).deliverable,
+  },
+];
+
+export default function ListingDetails({
+  listingData,
+}: {
+  listingData: Listing;
+}) {
+  const listingFields = () => {
+    if (listingData.listing_type === 'CASE') {
+      if (listingData.needs_interpreter) {
+        return caseInterpretationFields;
+      }
+      // will we ever need to render the case listing (instead of case interp) for a case that has needs_interpreter === true?
+      return caseFields;
+    }
+    if (listingData.listing_type === 'INT') {
+      return intepretationFields;
+    }
+    if (listingData.listing_type === 'DOC') {
+      return docFields;
+    }
+    return LCAFields;
+  }
+
+  const dateComponent = () => {
```

Comment:
looks like eslint is giving an error that we should probably fix: 
<img width="624" alt="image" src="https://github.com/calblueprint/immigration-justice-project/assets/84427964/637378bb-2a65-44c8-bcd8-57ff0605c47e">

Reply:
yes! fixed